docs/reference/java-interop.md

```diff
@@ -333,15 +356,14 @@ if (Character.isLetter(a)) {
 ### Java Reflection
 
 Java reflection works on Kotlin classes and vice versa. As mentioned above, you can use `instance.javaClass` or 
-`javaClass<ClassName>()` to enter Java reflection through `java.lang.Class`. You can then "convert" to Kotlin reflection
+`ClassName::class.java` to enter Java reflection through `java.lang.Class`. You can then "convert" to Kotlin reflection
 by calling `.kotlin`:
  
 ``` kotlin 
 val kClass = x.javaClass.kotlin  
 ```
  
-In much the same way you can convert from Kotlin reflection to Java: `ClassName::class.java` is the same as `javaClass<ClassName>()`.
-Other supported cases include acquiring a Java getter/setter method or a backing field for a Kotlin property, 
+Other supported cases include acquiring a Java getter/setter method or a backing field for a Kotlin property,
 getting a containing `KPackage` instance for a Java class, and getting a `KProperty` for a Java field.
```

Comment:
Not related to your changes, but I don't understand the part about `KPackage` and I think it should be removed...